identity/app/idapiclient/IdApi.scala

```diff
@@ -173,11 +187,33 @@ abstract class IdApi(val apiRootUrl: String, http: Http, jsonBodyParser: JsonBod
   def deleteTelephone(auth: Auth): Future[Response[Unit]] =
     delete("user/me/telephoneNumber", Some(auth)) map extractUnit
 
+  def deleteAccount(auth: Auth, emailPassword: EmailPassword): Future[Response[Unit]] =
+    delete("user/me", Some(auth), body = Some(write(emailPassword))) map extractUnit
+
   // THIRD PARTY SIGN-IN
   def addUserToGroup(groupCode: String, auth: Auth): Future[Response[Unit]] = {
     post(urlJoin("user", "me", "group", groupCode), Some(auth)) map extractUnit
   }
 
+  // Membership & Subscription
+  def userIsMember(auth: Auth): Future[Boolean] = {
+    http.GET(
```

Comment:
The indentation is a bit out of whack on these, makes it hard to see what is args to GET and what is processing the response.